okhttp/src/main/java/io/grpc/okhttp/OkHttpChannelBuilder.java

```diff
@@ -271,8 +271,7 @@ SSLSocketFactory createSocketFactory() {
       case TLS:
         try {
           if (sslSocketFactory == null) {
-            SSLContext sslContext = SSLContext.getInstance("TLS", Platform.get().getProvider());
-            sslContext.init(null, null, null);
+            SSLContext sslContext = SSLContext.getInstance("Default", Platform.get().getProvider());
```

Comment:
This is unexpected. I don't see any documentation for Default. It's not here, for instance: http://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html#SSLContext